src/style.css

```diff
@@ -0,0 +1,40 @@
+body {
+  margin: 0;
+}
+
+.star-container {
+  display: flex;
+  width: 100%;
```

Comment:
your container is a `div`, its `display` prop is block, so the width is 100% by default